cmd/goal/clerk.go

```diff
@@ -123,6 +125,7 @@ func init() {
 
 	compileCmd.Flags().BoolVarP(&disassemble, "disassemble", "D", false, "disassemble a compiled program")
 	compileCmd.Flags().BoolVarP(&noProgramOutput, "no-out", "n", false, "don't write contract program binary")
```

Comment:
@algochoi Unrelated to the line and for my understanding:  Since it's my 1st time closely reviewing a PR modifying `goal`, I'm wondering what manual steps you're referencing in the PR description (reproduced below)?

> This also changes the goal clerk compile command, so any manual changes to documentation needs to be done as well.

Reply:
I know of this one: https://github.com/algorand/docs/blob/staging/docs/clis/goal/clerk/compile.md

I think changing that should change the developer docs.

Reply:
@algochoi Gotcha - A few follow ups:
* I've seen in other places that go-algorand generates docs that are copied into _docs_ repo.  Is the linked doc generated by go-algorand or docs?
  * From a quick search, it seems like the linked doc is _not_ in go-algorand.
* Assuming the doc is maintained in docs, what is the protocol for making the update?  Trying to gauge if you'll open a PR or if there's a different workflow.

Reply:
There is a script in the docs folder that generates the docs from `goal`: https://github.com/algorand/docs/blob/fa7c454823c3ce6534f2cca6f1c79d538437f6a3/scripts/reformat-all-commands.sh#L8 

But it looks like we should nudge someone from DevRel to run it with the release.

Reply:
@algochoi Got it - thanks.  Presumably, we'd want to time the docs update with a release.  I added an issue comment to circle back later:  https://app.zenhub.com/workspaces/algorand-main-workspace-5ea71fda6d554d53dda36f16/issues/algorand/go-algorand/3725#issuecomment-1104090029.

Resolving.